pages/ksphere/konvoy/latest/install/install-aws/advanced-provisioning/index.md

```diff
@@ -265,10 +270,12 @@ spec:
   provider: aws
   aws:
     vpc:
-      enableVPCEndpoints: false
+      enableVPCEndpoints: true
 ...
 ```
 
+<p class="message--note"><strong>NOTE: </strong>When using a custom VPC with these endpoints already present, you should leave the value set to <code>enableVPCEndpoints: false</code>, otherwise Konvoy will modify the existing resources which could prevent your other workloads from accessing the AWS api.</p>
```

Comment:
```suggestion
<p class="message--note"><strong>NOTE: </strong>When using a custom VPC with these endpoints already present, you should leave the <code>enableVPCEndpoints: false</code> value set. Otherwise, Konvoy modifies existing resources which could prevent other workloads from accessing the AWS api.</p>
```